src/mdbExtensionController.ts

```diff
@@ -160,6 +160,15 @@ export default class MDBExtensionController implements vscode.Disposable {
     this._editorsController.registerProviders();
   }
 
+  subscribeToConfigurationChanges(): void {
+    const subscription = vscode.workspace.onDidChangeConfiguration((event) => {
+      if (event.affectsConfiguration('mdb.presetSavedConnections')) {
+        void this._connectionController.loadSavedConnections();
+      }
+    });
+    this._context.subscriptions.push(subscription);
```

Comment:
If I understand correctly, if I add this subscription to the context, it'll auto-dispose right so no further manual handling needed? @alenakhineika

Reply:
This is how I understand this as well.